service/templates/cloudformation/guest/outputs.yaml

```diff
@@ -1,9 +1,13 @@
 {{define "outputs"}}
 Outputs:
-  WorkersOutput:
+  MasterImageID:
+    Value: {{ .MasterImageID }}
+  MasterCloudConfigVersion:
+    Value: {{ .MasterCloudConfigVersion }}
+  Workers:
```

Comment:
I think WorkerCount would be a clearer name?

Reply:
Much better, done.